tests/api_workflow/utils.py

```diff
@@ -0,0 +1,8 @@
+import random
+
+# [a-f0-9]
+_CHARACTER_SET = [str(i) for i in range(10)] + [chr(i + 97) for i in range(6)]
```

Comment:
```suggestion
_CHARACTER_SET = "abcdef0123456789"
```
Might be a bit easier to understand :)